src/opni_nats/nats_wrapper.py

```diff
@@ -48,7 +58,11 @@ async def reconnected_cb():
             "max_reconnect_attempts": -1,
             "reconnect_time_wait": 5,
             "verbose": True,
+            "user": self.NATS_USERNAME,
+            "password": self.NATS_PASSWORD,            
```

Comment:
does NATS connect() complain if we pass these two fields and they're `None` types?

Reply:
Nope, they're already defaulted to None in the nats connect method - https://github.com/nats-io/nats.py/blob/master/nats/aio/client.py#L256-L257